src/main/java/LottoController.java

```diff
@@ -0,0 +1,29 @@
+import domain.Cash;
+import domain.LottoGame;
+import domain.LottoNumber;
+import domain.LottoTicket;
+import domain.LottoTickets;
+import domain.LottoWinningNumbers;
+
+public class LottoController {
+	private final LottoView view;
+
+	public LottoController(LottoView view) {
+		this.view = view;
+	}
+
+	public void play() {
+		Cash cash = view.getCash();
+		view.displayPaidTicketCount(cash.getTicketCount());
+
+		LottoTickets lottoTickets = LottoTickets.ofRandom(cash.getTicketCount());
```

Comment:
cash.`getTicketCount()`  연산이 상대적으로 간단한 연산이기는하지만 이미 수행한 기능을 또 다시 수행할 필요는 없을 것 같아요. 🙂 아래와 같이 변수로 추출하여 재사용하도록 변경해볼 수 있을 것 같아요.

```suggestion
                int countOfTickets = cash.getTicketCount();
		view.displayPaidTicketCount(countOfTickets);

		LottoTickets lottoTickets = LottoTickets.ofRandom(countOfTickets);
```

Reply:
반영하였습니다.
https://github.com/next-step/java-lotto-kakao/pull/14/commits/b762854b9d2ec3df442652e7d96ee048a40f4cdb